src/containers/datasets/national-dashboard/widget.tsx

```diff
@@ -0,0 +1,91 @@
+import Loading from 'components/loading';
+import { WIDGET_CARD_WRAPPER_STYLE, WIDGET_SUBTITLE_STYLE } from 'styles/widgets';
+
+import { COLORS } from './constants';
+import { useNationalDashboard } from './hooks';
+import IndicatorSources from './indicator-sources';
+import OtherResources from './other-resources';
+
+const FAKE_DATA = {
+  data: [
+    {
+      indicator: 'floods',
+      sources: [
+        {
+          source: 'voluptatem',
+          unit: 'GDP',
+          years: [2016],
+          data_source: [
+            {
+              year: 2016,
+              value: 68950.02,
+              layer_info: 'Enim repellat pariatur est.',
+              layer_link: 'http://kutch-spencer.com/moises',
+              download_link: 'http://kutch-spencer.com/moises',
+            },
+          ],
+        },
+      ],
+    },
+  ],
+  metadata: {
+    location_id: '226',
+    other_resources: [
+      {
+        name: 'voluptatem',
+        description: 'Enim repellat pariatur. Earum modi eos. Libero tempora exercitationem.',
+        link: 'http://kutch-spencer.com/moises',
+      },
+    ],
+    note: null,
+  },
+};
+const NationalDashboard = () => {
+  const { data, isLoading, isFetching, isFetched } = useNationalDashboard();
+  const { data: dataIndicators, metadata } = FAKE_DATA;
```

Comment:
Why do you need to call here `FAKE_DATA`? You should be able to access the fake data through the `data` property. The idea is to have the mock in one place and not spread it so it's easier to change later.